src/external_registration.cpp

```diff
@@ -69,7 +81,7 @@ int main(int argc, char** argv)
 
   // everything's correctly parsed - let's start the app!
   app.start();
-  boost::shared_ptr<naoqi::Driver> bs = qi::import("naoqi_driver_module").call<qi::Object<naoqi::Driver> >("ROS-Driver", app.session(), vm["namespace"].as<std::string>()).asSharedPtr();
+  boost::shared_ptr<naoqi::Driver> bs = qi::import("naoqi_driver_module").call<qi::Object<naoqi::Driver> >("ROS-Driver", app.session(), vm["node_name"].as<std::string>()).asSharedPtr();
```

Comment:
oh right, as it is a default argument. How about using the proper ROS API then: http://docs.ros.org/jade/api/roscpp/html/namespaceros_1_1this__node.html